scripts/distribution/macOS-package/template/scripts/postinstall

```diff
@@ -17,6 +17,27 @@ function addBinariesToPath() {
     ln -f -s "/Library/Concordium Node/node-collector" "$symlinkCollector"
 }
 
+# Replaces all occurrences of 'placeholder' with 'replacement' in file 'fileName'.
+# The replacement can contain any characters except for special characters in XML, which must be escaped.
+function replaceInFile() {
+    local placeholder=$1
+    local replacement=$2
+    local fileName=$3
+
+    # Other tools, such as sed and awk, has limitations to the characters allowed, so '/' and '&' could not be used.
+    # This tiny script does not have that problem.
+    python3 - <<EOF
```

Comment:
Is python3 always installed globally on macs?

Reply:
I believe python 2.7.x is installed by default. Python 3 should probably be fetched from homebrew etc.

Reply:
Then we should use the version that is definitely installed.